package.json

```diff
@@ -315,7 +316,6 @@
     "google-closure-library": "^20200112.0.0",
     "html2canvas": "=1.0.0-rc.4",
     "jquery": "3.4.1",
```

Comment:
Since it's wasm, I'm sure it's fine, but please resolve this merge and double-check that jQuery 3.5 doesn't break anything for the chardetng-wasm.

Reply:
It does not.

Reply:
I fixed the merge conflict instead. And yup, I'd be more than a little surprised if _any_ wasm library had a dependency on jquery.